components/Navbar.tsx

```
@@ -0,0 +1,29 @@
+"use client";
+import React from "react";
+
+const Navbar: React.FC = () => {
+  return (
+    <header className="flex justify-between items-center px-6 py-0 h-16 bg-neutral-800 max-sm:px-4 max-sm:py-0">
+      <div className="h-6 w-[117px]">
+        <div
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)